lib/localstorage.dart

```diff
@@ -1,17 +1,18 @@
-import 'dart:io';
 import 'dart:async';
 import 'dart:convert';
+
 import 'package:flutter/foundation.dart' show ValueNotifier;
-import 'package:path_provider/path_provider.dart';
+
+import 'src/directory/directory.dart';
```

Comment:
Use `package:` here

Reply:
The reason why you have to use this is so it works on the web. You need conditional imports so it pulls in the right code at compile time

Reply:
See other comment. You should use the package-style import